app/src/pages/inside/testPlansPage/milestones/milestonesTable/milestoneCard/milestoneCard.scss

```
@@ -0,0 +1,232 @@
+      color: var(--rp-ui-color-primary);
+      background: transparent;
+      border: 1px solid var(--rp-ui-color-primary);
+
+      &:hover {
+        background: var(--rp-ui-base-bg-200);
+      }
+    }
+  }
+
+  &__status-toggle_completed &__status-chevron {
+    color: currentColor;
```

Stylelint (17.4.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)